benchmarks/benchmark.c

```diff
@@ -73,9 +76,13 @@ void random_bignum(uint64_t k,uint64_t *a)
 
 // Core repetition of the function being benchmarked. The function
 // is recomputed on the same inputs each time without trying to
-// enforce serial dependency.
+// enforce serial dependency. The default is "repeat", but there
+// are variants doing either more or fewer reps by a given factor,
+// rounded up a bit in the latter case to at least 2.
 
-#define repeat(bod) {int i; for (i = 0; i < inner_reps; ++i) bod; }
+#define repeat(bod) {inner_reps = default_reps; int i; for (i = 0; i < inner_reps; ++i) bod; }
+#define repeatmore(n,bod) {inner_reps = n*default_reps; int i; for (i = 0; i < inner_reps; ++i) bod; }
+#define repeatfewer(n,bod) {inner_reps = (default_reps/n)+2; int i; for (i = 0; i < inner_reps; ++i) bod; }
```

Comment:
Is there a reason why +2 was added to `inner_reps`? This might not be a very important question, I am simply curious about the reason.